### ISSUE TITLE
Switch ui-kit to mosaic

### ISSUE THREAD

**Issue by philsturgeon** [stoplightio/json-schema-viewer#115]
## Summary

Switch from ui-kit to equivalent mosaic components.

## Tasks

- [x] Make sure all unprefixed Tailwind class names are gone
- [x] Remove all trace of Blueprint CSS
- [ ] Delete any dependencies relating to ui-kit

**Comment by marcelltoth** [comment]
FYI there's nothing directly from ui-kit in JSV. The only reason we have it listed is because it's a peer of markdown viewer. 

This should be blocked by the MDV issue, as it's non-actionable until that's gone.

**Comment by philsturgeon** [comment]
Yep!

**Comment by Fox32** [comment]
I thought about using `json-schema-viewer` in Backstage and had a look at the beta releases because of the extended support for newer schema versions. But I noticed that is depends on mosaic which is not open source and doesn't have a license on NPM. Any plans to open source that package?

**Comment by philsturgeon** [comment]
Is mosaic blocking you from installing the repo or is it just a legal/licensing concern?

**Comment by Fox32** [comment]
A legal/licensing concern. Otherwise it works and looks great - theming was a bit difficult with no direct access to the code or docs.

**Comment by philsturgeon** [comment]
@Fox32 thanks for letting me know, easy solve: mosaic now contains the Apache license. We're going to tag a new version that makes that clear on NPM, but here you go: 

<img width="1290" alt="image" src="https://user-images.githubusercontent.com/67381/113563474-aa391980-95ff-11eb-8a4d-6e33f71c2e0f.png">

**Comment by Fox32** [comment]
Thanks, looking forward to use the package!

**Comment by Fox32** [comment]
Oh, maybe I was a bit eager with using the new version. Looks like moasic doesn't play nicely when getting integrated into existing applications that aren't based on mosaic. CSS is applied globally and is overwriting our CSS rules. Would be cool if there is a way to limit the scope, but I couldn't have a deeper look into it.

**Comment by philsturgeon** [comment]
@Fox32 sorry for the trouble. Just to set expectations, we're not really expecting too many folks to use json-schema-viewer directly at the moment. We've not put the time into documenting it, or any of the usual work we normally do with our supported OSS projects like Elements, Prism, or Spectral. 

There should be a way to limit the scope though as Elements will need to be embedded into various existing applications and we don't want to be messing things up.

@marcelltoth do you have any tips or ideas for @Fox32?

**Comment by philsturgeon** [comment]
Alright just checked an issue in mosaic and it’s gonna happen, just not sure when. Weeks or months not years I expect. Thanks for the feedback @Fox32!

**Comment by Fox32** [comment]
Thanks for filling an issue 👍 I totally understand that this is an early state, so my expectations aren't that strong yet.

**Comment by marbemac** [comment]
@Fox32 I've split the global css reset concern over to a separate issue that we can track - #143.

**Comment by Fox32** [comment]
> @Fox32 I've split the global css reset concern over to a separate issue that we can track - #143.

Right, that doesn't really belong here, thanks!
